sqlgen/db.go

```diff
@@ -33,9 +36,20 @@ func (db *DB) Query(ctx context.Context, result interface{}, filter Filter, opti
 	if err != nil {
 		return err
 	}
-	querySql, fields := query.ToSQL()
+	clause, fields := query.ToSQL()
 
-	res, err := db.QueryExecer(ctx).Query(querySql, fields...)
+	if span := opentracing.SpanFromContext(ctx); span != nil {
```

Comment:
I refactored these lines a little to a point that I am okay with. I can't think of good ways to make this more succinct, since we need to both replace the existing `ctx`, and also call `defer` in this function.